src/zkevm_specs/state.py

```diff
@@ -267,7 +269,13 @@ def check_call_context(row: Row, row_prev: Row):
 @is_circuit_code
 def check_account(row: Row, row_prev: Row, row_next: Row, tables: Tables):
     get_addr = lambda row: row.address()
-    get_field_tag = lambda row: row.field_tag()
+
+    field_tag = row.field_tag()
+    proof_type = MPTProofType.IsNonceMod  # type warning if None
```

Comment:
I think it would be better if we can match the values of `MPTProofType` with their corresponding cases in `AccountFieldTag`, so that we don't need to perform conversions in the circuit.
For example we could have:
```python
class AccountFieldTag(IntEnum):
    Nonce = 1
    Balance = 2
    CodeHash = 3

class MPTProofType(IntEnum)
    IsNonceMod = 1
    IsBalanceMod = 2
    IsCodeHashProof = 3
    IsStorageMod = 4
    IsAccountDeleteMod = 5
    IsNonExistingAccountProof = 6
```